parquet-scrooge/src/main/java/org/apache/parquet/scrooge/ScroogeReadSupport.java

```diff
@@ -37,8 +38,14 @@ public ScroogeReadSupport() {
   }
 
   @Override
+  protected MessageType getProjectedSchema(Configuration configuration, FieldProjectionFilter fieldProjectionFilter) {
+    ThriftType.StructType thriftStruct = new ScroogeStructConverter().convert(thriftClass);
+    return new ThriftSchemaConverter(configuration, fieldProjectionFilter).convert(thriftStruct);
+  }
+
+  @Deprecated
   protected MessageType getProjectedSchema(FieldProjectionFilter fieldProjectionFilter) {
     ThriftType.StructType thriftStruct = new ScroogeStructConverter().convert(thriftClass);
```

Comment:
thriftStruct is an unused variable, the result of this conversion is not used, could you please delete this?

Reply:
My bad, this is now removed.